xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -5046,6 +5046,18 @@ core.rename.newspace.hint=Containing space for the renamed page
 core.rename.newpage=New page
 core.rename.newpage.hint=Name of the renamed page
 
+#######################################
+## until 7.2M3
+#######################################
+
+### Dashboard
+
+#@deprecated platform.dashboard.space.documents
```

Comment:
See http://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HTranslationPropertyDeprecations

#@deprecated new.key.name
old.key.name=

Currently the keys are identical